command/helper/helper.go

```diff
@@ -30,6 +31,18 @@ func (r *ClientCloseResult) GetOutput() string {
 	return r.Message
 }
 
+type IPBindings struct {
+	Localhost     string
+	AllInterfaces string
+}
+
+var (
+	IPBinding = IPBindings{
+		Localhost:     "127.0.0.1",
+		AllInterfaces: "0.0.0.0",
+	}
+)
+
```

Comment:
Why not make this an enum?

Doesn't need to be a global var struct, just a global enum is fine 🙂 

```go
type IPBinding string

const (
   LocalHostBinding IPBinding = "127.0.0.1",
   AllInterfacesBinding IPBinding = "0.0.0.0"
)
```

Then modify the `ResolveAddr` to take IPBinding instead of string

Reply:
Thank you for this suggestion 🙂 Implemented as enum b4b23dca61a8cfea98ef5fd27c470cc1abb05dc8